PyU4V/provisioning.py

```diff
@@ -1593,6 +1595,9 @@ def create_storage_group(
                      'workloadSelection': workload,
                      'volumeAttributes': [volume_attributes]}
 
+        if enable_mobility_id:
+            slo_param.update({'enable_mobility_id': enable_mobility_id})
```

Comment:
I take it that the REST call happily takes True boolean as well as 'true' string?

Reply:
it does, I've tested this way no issue